client/dashboard/utils/site-features.ts

```diff
@@ -0,0 +1,49 @@
+// client/dashboard/sites/utils/site-features.ts
+import type { Site } from '../data/types';
+
+/**
+ * Centralized place for site feature checks
+ * This helps maintain consistency and makes it easier to update feature detection logic
+ */
+
+export const SiteFeatures = {
```

Comment:
I think we can just export all the functions individually from this file, rather than grouping them into a single object.

Reply:
👍 

I'll wait for other feedback and refactor afterward.

Reply:
I agree; I don't see any benefit of having all this in an object; we always use a tiny slice of them in individual components.